common/src/main/java/tds/common/logging/EventLogger.java

```diff
@@ -88,8 +88,8 @@ public Object getField(final String key) {
      * @param message    an optional free-format message string to add to the log entry
      * @param data       optional searchable event data to add to the recorded event
      */
-    public void info(final String app, final String logEvent, final String checkpoint,
-                     final String message, final Map<EventData, Object> data) {
+    public void trace(final String app, final String logEvent, final String checkpoint,
```

Comment:
Why not just call this `log()`?  The implementation can log at a trace level, but that way if we decided we'd want event logging in the future to be at a different level we wouldn't have to go and change all these classes.  It seems like the clients of the EventLogger don't need to know the level the event logs are logged at just that they're logged.